tests/KernelFlexibleCenterer_tests.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+from skcosmo.preprocessing.KernelFlexibleCenterer import KernelFlexibleCenterer
+import sklearn as sk
+import numpy as np
+
+
+class TestStringMethods(unittest.TestCase):
```

Comment:
you should add doc for each of these tests as to what the "expected" behavior is and why